### PR TITLE
package google-compute-engine

### PR DESCRIPTION
**Pull request by elvirag**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dbast** [comment]
There are now build requirements... can this be `noarch: python`? (requires also the skip for windows to be removed)

**Comment by elvirag** [comment]
I don't know what you mean by "There are no build requirements", but the package is not supposed to run on Windows, because it works on posix only, and has no meaning in windows

File: recipes/google-compute-engine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    - google_metadata_script_runner --help
+
+about:
+  home: https://github.com/GoogleCloudPlatform/compute-image-packages
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Google Compute Engine
+  doc_url: https://cloud.google.com/compute/docs/
+  dev_url: https://pypi.io/packages/source/g/google-compute-engine/google-compute-engine-2.8.13.tar.gz
```
Comment (chrisburr) [inline recipes/google-compute-engine/meta.yaml]: This should point to the repository/webpage used for development:
```suggestion
  dev_url: https://github.com/GoogleCloudPlatform/compute-image-packages/tree/master/packages/python-google-compute-engine
```

File: recipes/google-compute-engine/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  commands:
+    - google_accounts_daemon --help
+    - google_clock_skew_daemon --help
+    - google_instance_setup --help
+    - google_network_daemon --help
+    - google_metadata_script_runner --help
+
+about:
+  home: https://github.com/GoogleCloudPlatform/compute-image-packages
+  license: Apache Software
```
Comment (chrisburr) [inline recipes/google-compute-engine/meta.yaml]: Conda forge generally prefers the SPDX abbreviations:
```suggestion
  license: Apache-2.0
```

File: recipes/google-compute-engine/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+requirements:
+  host:
+    - boto
+    - distro
+    - pip
+    - python
+    - setuptools
+  run:
+    - boto
+    - distro
```
Comment (chrisburr) [inline recipes/google-compute-engine/meta.yaml]: According to [the `setup.py`](https://github.com/GoogleCloudPlatform/compute-image-packages/blob/master/packages/python-google-compute-engine/setup.py#L24-L25):
```suggestion
    - distro  # [py>=36]
```

File: recipes/google-compute-engine/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    - google_clock_skew_daemon=google_compute_engine.clock_skew.clock_skew_daemon:main
+    - google_instance_setup=google_compute_engine.instance_setup.instance_setup:main
+    - google_network_daemon=google_compute_engine.networking.network_daemon:main
+    - google_metadata_script_runner=google_compute_engine.metadata_scripts.script_manager:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
+
+requirements:
+  host:
+    - boto
```
Comment (chrisburr) [inline recipes/google-compute-engine/meta.yaml]: Are these all needed in `host`? I suspect you only need `python`, `pip`.

Comment (elvirag) [reply recipes/google-compute-engine/meta.yaml]: I'll check

Comment (elvirag) [reply recipes/google-compute-engine/meta.yaml]: Yes, they are needed.

**Pull request merged**